docs/release-notes/8.4.asciidoc

```diff
@@ -1,6 +1,20 @@
 [[release-notes-header-8.4.0]]
 == 8.4
 
+[discrete]
+[[release-notes-8.4.2]]
+=== 8.4.2
+
+[discrete]
+[[bug-fixes-8.4.2]]
+==== Bug fixes and enhancements
+* Removes access to the **Notes** and **Pinned** tabs in Timeline templates ({pull}140478[#140478]).
```

Comment:
Do we need to update any other docs, if this removes functionality in the Timeline templates page?

Reply:
Good catch. Just a couple screenshots in this topic: https://www.elastic.co/guide/en/security/master/timeline-templates-ui.html#create-timeline-template

Reply:
Addressing this in https://github.com/elastic/security-docs/issues/2467.